app/index.js

```diff
@@ -86,6 +87,12 @@ module.exports = yeoman.Base.extend({
 					message: 'A short description of your project:'
 				},
 				{
+					type: 'confirm',
+					name: 'banner',
+					message: 'Do you like to use a banner in your production files?',
+					default: false
```

Comment:
Wouldn’t it make sense to set the additional property `store: true` to store that value for future use of that generator?